cypress/integration/signin_authentication/forgot_password_spec.js

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../fixtures/users.json';
+
+import {reUrl} from '../../utils';
+
+const user1 = users['user-1'];
```

Comment:
For this I'd create a new user, this feels too risky if something happens in this test and that user now has an invalid password and it can impact downstream tests.

Reply:
Totally agree, now updated.